ProjectBank/Infrastructure.Tests/UserRepositoryTests.cs

```diff
@@ -0,0 +1,21 @@
+namespace ProjectBank.Infrastructure.Tests;
+public class UserRepositoryTests : TestSetup, IDisposable
+{
+    [Fact]
+    public void testExample()
+    {
+        var testUser = new User("Jeppe", "korg@itu.dk", "test", Role.Student);
```

Comment:
We need to take a choice on wheather we add data here in the test cases or only in TestSetup's addTestData method. Otherwise the testdata (and tests) could be pretty  unpredictable when new testcases are added with new data.